src/sider/sykmeldinger/container/DinSykmeldingSide.tsx

```diff
@@ -63,7 +55,6 @@ const DinSykmeldingSide = (): ReactElement => {
     >
```

Comment:
Synes det var rart på screenshotten at sidetittelen hadde hvit bakgrun, for den pleier å være grå på de andre sidene, men jeg synes egentlig det ser bedre ut 👍🏼

Reply:
Aha, at den ligger innenfor "boksen" liksom? Hvis den ikke hadde gjort det så hadde vi ikke hatt en overskrift for boksen 🥲 Synes det hadde blitt litt rart

Reply:
Ja, jeg synes vi skal flytte alle andre overskrifter inn i boksen egentlig, men det blir jo en annen oppgave 🤷🏼‍♂️